src/sentry/api/serializers/models/release.py

```diff
@@ -325,12 +325,16 @@ def __get_release_data_no_environment(self, project, item_list):
 
         first_seen: dict[str, datetime.datetime] = {}
         last_seen: dict[str, datetime.datetime] = {}
-        tag_values = tagstore.backend.get_release_tags(
-            organization_id,
-            project_ids,
-            environment_id=None,
-            versions=[o.version for o in item_list],
-        )
+        if no_snuba_for_release_creation:
```

Comment:
question: why is snuba being called in a serializer? It doesn't make sense to me to have something that is supposed to serialize data make IO calls

Reply:
^ good point 🤔